portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -45,4 +36,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.setContentType("application/json;");
         response.getWriter().println(jsonComment);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String name = getParameter(request, "name", "");
+    String comment = getParameter(request, "comment", "");
+
+    // Respond with the result.
+    comments.add(name + ":\n" + comment);
```

Comment:
Using a newline to separate fields is rather ineffective when you're going to end up rendering to HTML, unless you later parse it back.

Consider instead storing the comments as an ArrayList<Comment>, where Comment is an AutoValue type with an author and comment (see https://github.com/google/auto/blob/master/value/userguide/index.md). Then in doGet() you can convert the comment to a JSON Object with the same structure, and - in the JavaScript - you can format each one separately (apply different formatting to author and text, etc.).

[As a more general rule: when possible, try to push all user-visible formatting to the UI, or as close to it as you can. In this case, your use of a colon+newline - a formatting choice - is now at the entry to the backend, which is too early.]

Reply:
I'll do it in the next PR